bitnami/minio/README.md

```diff
@@ -84,7 +84,11 @@ The following table lists the configurable parameters of the MinIO chart and the
 | `statefulset.podManagementpolicy`    | Statefulset pods management policy                                                                                                                        | `Parallel`                                              |
 | `deployment.updateStrategy`          | Deployment update strategy policy                                                                                                                         | `Recreate`                                              |
 | `existingSecret`                     | Existing secret with MinIO credentials                                                                                                                    | `nil`                                                   |
-| `useCredentialsFile`                 | Have the secret mounted as a file instead of env vars                                                                                                     | `false`                                                 |
+| `credentialsFile.enabled`            | Use credential files in stead of env vars                                                                                                     | `false`                                                 |
```

Comment:
```suggestion
| `credentialsFile.enabled`            | Use credential files instead of env vars                                                                                                      | `false`                                                 |
```